docs/content/api-overview/resources/web-app.md

```diff
@@ -56,7 +56,7 @@ The Web App builder is used to create Azure App Service accounts. It abstracts t
 | Web App | add_slot | Adds a deployment slot to the app |
 | Web App | add_slots | Adds multiple deployment slots to the app |
 | Web App | health_check_path | Sets the path to your functions health check endpoint, which Azure load balancers will ping to determine which instances are healthy.|
-| Web App | custom_domain | Adds custom domain to the app, containing an app service managed certificate |
+| Web App | custom_domain | Adds a custom domain to the app.  Optionally this can also generate an SSL certificate for the domain. This operator can be used multiple times to add multiple domains |
```

Comment:
Could you provide an example or simple sentence as to how to generate the SSL cert?

Reply:
I've added a bit more details into the description here. Is this sufficient or should I add an example further down as well?